### PR TITLE
OU-167: Show Query button is in "Streaming logs" status. 

### PR DESCRIPTION
**Pull request by zhuje**
**Related Issue** 
https://issues.redhat.com/browse/OU-167

**Overview** 
Updated loki-client so that the websocket remains open for logs streaming. The issue was because of a malformed websocket URL. Also updated the script start-console.sh to remove the existing env file before creating a new one.

**Explanation** 
Logs streaming begins by displaying logs from the past hour. Previously, we set this to `start=3600000` milliseconds (i.e. 1 hour) but the Loki HTTP API docs state this parameter should be in **nanoseconds Unix epoch** (e.g. `start=1685563694`, meaning May 31, 2023, 8:8:14PM GMT). Because we were not formatting the websocket url correctly we could not stream the logs. 

This PR removes the ‘start’ parameter because the Loki HTTP API already defaults the 'start' parameter to the past hour, ”`start`: The start time for the query as a nanosecond Unix epoch. Defaults to one hour ago.” (source: https://grafana.com/docs/loki/latest/api/#stream-log-messages). 

image

**Before** 
Websocket can not stream any logs and closes immediately after opening. 
[OU-167](https://issues.redhat.com//browse/OU-167) BEFORE

**After**
Websocket can now stream logs. 
image

### CODE REVIEW

File: web/package-lock.json
Comment (kyoto) [inline web/package-lock.json:1]: Are these `package-lock.json` related to this change? If not let's keep any package updates in a separate PR.

File: web/src/loki-client.ts [hunk 1]
```diff
@@ -26,6 +26,19 @@ type HistogramQuerParams = {
   tenant: string;
 };
 
+type LokiTailQueryParams = {
```
Comment (kyoto) [inline web/src/loki-client.ts:29]: I notice that `LokiTailQueryParams` isn't used. Could you briefely explain the reason for separating out `LokiTailQueryParams` and `CustomTailQueryParams`?

File: web/scripts/start-console.sh
Comment (kyoto) [inline web/scripts/start-console.sh:1]: I think this change is unrelated to the streamling logs fix? If so, probably better to break it into a separate PR.

**Review by kyoto — CHANGES_REQUESTED** [review]

File: web/package-lock.json
Comment (zhuje) [reply web/package-lock.json:1]: No, they're not related to the bug fix. I ran 'make install' in the logging-view-plugin and it updated the `package-lock.json`. I'll remove it from this PR.

File: web/src/loki-client.ts (hunk 1 above)
Comment (zhuje) [reply web/src/loki-client.ts:29]: So the LokiTailQueryParms parameters are the ones Loki HTTP API requires and the other parameters are ones that the plugin requires. But you're right it might be less confusing to just make them into a single type.

File: web/scripts/start-console.sh
Comment (zhuje) [reply web/scripts/start-console.sh:1]: Yeah, you're right. I'll update this.

**Review by kyoto — APPROVED** [review]
Thanks @zhuje!
